build.rs

```diff
@@ -2,22 +2,35 @@ use std::process::Command;
 
 // Example custom build script.
 fn main() {
+    // TODO: use handlebars to automatically generate filters, that way you abstract away the code that is used
+    //       for all filters
+
     // Note: needs to be run with cargo -vv to generate output from these commands.
     eprintln!("{:?}",
-              Command::new("rustc").args(&["plugins/sample.rs", "--crate-type=dylib"]).output().unwrap());
+              Command::new("rustc").args(&["plugins/sample_filter.rs", "--crate-type=dylib"]).output().unwrap());
+    /*
     eprintln!("{:?}",
               Command::new("rustc").args(&["plugins/sample2.rs", "--crate-type=dylib"]).output().unwrap());
+    eprintln!("{:?}",
+              Command::new("rustc").args(&["plugins/compiled_sample.rs", "--crate-type=dylib"]).output().unwrap());
     eprintln!("{:?}",
               Command::new("rustc").args(&["plugins/sink.rs", "--crate-type=dylib"]).output().unwrap());
+    */
 
     // Don't go any further if these commands failed.
-    assert!(Command::new("rustc").args(&["plugins/sample.rs",  "--crate-type=dylib"]).status().unwrap().success());
+    assert!(Command::new("rustc").args(&["plugins/sample_filter.rs",  "--crate-type=dylib"]).status().unwrap().success());
+    /*
```

Comment:
Similar comment as above. Let's remove lines of code that are commented out if they are no longer needed.

Reply:
Good call.  This is fixed now.